himportscan/exe/Main.hs

```diff
@@ -5,11 +5,14 @@ module Main where
 import HImportScan.ImportScanner (ScannedImports, scanImportsFromFile)
 import qualified Data.Aeson as Aeson
 import qualified Data.ByteString.Lazy.Char8 as ByteString.Lazy
+import Data.Maybe (catMaybes)
 
 main :: IO ()
 main = do
   haskellSrcs <- lines <$> getContents
-  mapM scanImportsFromFile haskellSrcs >>= printImports
+  -- silently ignore missing files
+  -- TODO: it would be better to instead report them to the user
```

Comment:
I would do this here, but I would need to alter the JSON format with which go and hs communicate, so I'd prefer this to be done in a separate PR.